esp/eclwatch/ws_XSLT/nls/bs/hpcc.xml

```diff
@@ -155,6 +155,7 @@
 <st id='ViewDetails'>Pogledajte Detalje</st>
 <st id='ViewingPage'>Pregled stranice</st>
 <st id='ViewLogFile'>Pogledajte Log Datoteku</st>
+<st id='ViewAuditLogFile'>View Audit Log File</st>
```

Comment:
@wangkx Do we have to update these foreign translation files with English now?  I thought we could leave these off and english would be used until these files were updated?  My understanding could be wrong or outdated though, so I'm just asking.

Reply:
When I was asking to do Chinese translation for 6.0, the Chinese translation file did have the items with English. My guess is I have to update these foreign translation files with English now so that someone may translate later.  @lorraineachapman is my guess correct?

Reply:
@GordonSmith Can you comment?

Reply:
For ECL Watch we add all new (English) terms to:  https://github.com/hpcc-systems/HPCC-Platform/blob/master/esp/src/eclwatch/nls/hpcc.js and then prior to the gold release, we send out lists of new terms that need to be translated and update in the one go.

Reply:
I suspect adding untranslated items to the foreign files is going to complicate the process of sending out the new terms - best not to!